### PR TITLE
ingress is supported

### PR DESCRIPTION
**Pull request by linki**
state that we now support ingress

### CODE REVIEW

**Comment by Raffo** [comment]
👍

File: README.md [hunk 1]
```diff
@@ -36,11 +36,11 @@ $ mate \
     --aws-record-group-id=foo
 ```
 
-For each exposed service Mate will create two records in Route53: 
+For each exposed service Mate will create two records in Route53:
 
-1. A record - Alias to the ELB with the name inferred from `kubernetes-format` and `kubernetes-domain`. So if you create an nginx service named `my-nginx` in `default` namespace and domain `example.com` registered record will have a hostname `default-my-nginx.example.com`. 
- 
-2. TXT record - which will have the same name as an A record (`default-my-nginx.example.com`) and a special identifier with embedded `aws-record-group-id` value. TXT record helps to identify which records are created via Mate and makes it safe not to overwrite manually created records 
+1. A record - An Alias to the ELB with the name inferred from `kubernetes-format` and `kubernetes-domain`. So if you create an nginx service named `my-nginx` in the `default` namespace and use a `example.com` as domain the registered record will have a hostname of `default-my-nginx.example.com`.
```
Comment (ideahitme) [inline README.md]: probably need to mention that we can use Annotation, which takes precedence.

File: README.md [hunk 2]
```diff
@@ -55,33 +55,32 @@ $ mate \
...
 
 ### Kubernetes
 
-By default Mate will retrieve the list of services from Kubernetes API server via `http://127.0.0.1:8001` (for local testing use `kubectl proxy`), however API server url can be configured with `kubernetes-server` flag. 
-Mate will listen for events from Kubernetes API Server and create corresponding records for newly created services. Further synchronization (create, removal, update) will occur every minute. If you only like to do the synchronization you can enable `sync-only` flag. 
+By default Mate will retrieve the list of services from the Kubernetes API server via `http://127.0.0.1:8001` (for local testing use `kubectl proxy`), however the API server url can be configured with the `kubernetes-server` flag.
+Mate will listen for events from the API Server and creates corresponding records for newly created services. Further synchronization (update and removal) will occur every minute. There's an initial syncronization when Mate boots up so it's safe to reboot the process at any point in time. If you only like to do the synchronization you can use the `sync-only` flag.
```
Comment (ideahitme) [inline README.md]: ` (update and removal) will occur every minute...` - i think create happens as well, just via upsert. This is especially relevant if `sync-only` is enabled.

**Comment by ideahitme** [comment]
gotta reconfigure zappr, i think right now we can +1 our own pull requests and merge them :P

**Comment by linki** [comment]
@ideahitme updated the docs based on your feedback

**Comment by ideahitme** [comment]
👍

**Pull request merged**